packages/web-app-files/src/components/SideBar/Shares/Collaborators/CollaboratorsEditOptions.vue

```diff
@@ -283,11 +281,7 @@ export default {
     },
 
     minExpirationDate() {
-      const date = new Date()
-
-      date.setDate(new Date().getDate() + 1)
-
-      return date
+      return DateTime.now().setLocale(this.$language.current).toJSDate()
```

Comment:
@kulmann this makes changes/rebase in #6039 necessary, too from my understanding